README.md

```diff
@@ -0,0 +1,45 @@
+Hm-admin
+--------
+
+[![Build Status](https://travis-ci.org/coopTilleuls/hm-admin.svg?branch=master)](https://travis-ci.org/coopTilleuls/hm-admin)
+[![Coverage Status](https://coveralls.io/repos/github/coopTilleuls/hm-admin/badge.svg?branch=master)](https://coveralls.io/github/coopTilleuls/hm-admin?branch=master)
+[![Issue Count](https://codeclimate.com/github/coopTilleuls/hm-admin/badges/issue_count.svg)](https://codeclimate.com/github/coopTilleuls/hm-admin)
+
+
+Hack with docker
+================
+
+To quickly start hacking, hm-admin comes with a docker ready development environment.
+
+It leverages three services:
+
+- `hm-admin`: The interesting part in our case. Hm-admin is running in this container and hm-admin sources are shared with the host
+- `api` and `mysql`: Those containers are responsible for the API to be developed and tested against. The API is made from https://api-platform.com/.
+
+
+## Initial steps for setting up your docker development environment
+
+First, you don’t want to recreate your database or redownload all node_modules each time you remove your containers so, we create two data containers:
+
+```
+docker create --name hmadmin_node_modules -v /app/node_modules tianon/true
+docker create --name hmadmin_mysql_data -v /var/lib/mysql mysql:5
+```
+
+Secondly, we need to install node dependencies
+
+```
+docker-compose run hm-admin npm install
```

Comment:
Yeah, we already have discussed this subject ;), next time we meet, we should settle for good this argument!

I’m not sure installing node modules on build, then mounting the project directory with a volume (then not using the node_modules installed on build)  see my all point here ;) https://github.com/api-platform/api-platform/pull/57#issuecomment-203903961